--- "/home/stephane/T\303\251l\303\251chargements/uvk5cec-0.3q/app/main.c"	2024-01-19 14:43:30.000000000 +0100
+++ app/main.c	2026-09-02 19:46:07.519502679 +0200
@@ -30,6 +30,12 @@
 #ifdef ENABLE_SPECTRUM
 #include "app/spectrum.h"
 #endif
+#ifdef ENABLE_SARSAT
+#include "app/sarsat.h"
+#endif
+#ifdef ENABLE_APRS
+#include "app/aprs.h"
+#endif
 
 #include "audio.h"
 #include "board.h"
@@ -204,7 +210,10 @@
 
 		case KEY_5:
 			if(beep) {
-#ifdef ENABLE_NOAA
+#ifdef ENABLE_APRS
+				APP_RunAprs();
+				gRequestDisplayScreen = DISPLAY_MAIN;
+#elif defined(ENABLE_NOAA)
 
 				if (!IS_NOAA_CHANNEL(gTxVfo->CHANNEL_SAVE))
 				{
@@ -245,8 +254,13 @@
 			break;
 
 		case KEY_8:
+#ifdef ENABLE_SARSAT
+			APP_RunSarsat();                 // F+8 : open the SARSAT screen
+			gRequestDisplayScreen = DISPLAY_MAIN;
+#else
 			gTxVfo->FrequencyReverse = gTxVfo->FrequencyReverse == false;
 			gRequestSaveChannel = 1;
+#endif
 			break;
 
 		case KEY_9:
