--- "/home/stephane/T\303\251l\303\251chargements/uvk5cec-0.3q/ceccommon.h"	2024-01-19 14:43:30.000000000 +0100
+++ ceccommon.h	2026-09-02 00:28:40.335925251 +0200
@@ -24,27 +24,27 @@
 #include <stdbool.h>
 #include <math.h>
 #include <stdio.h>
-#include "driver\bk4819.h"
-#include "driver\keyboard.h"
+#include "driver/bk4819.h"
+#include "driver/keyboard.h"
 #include "audio.h"
 #include "string.h"
 #include <stdint.h>
 #include <string.h>
 #include "external/printf/printf.h"
 //#include "gpsinfo.h"
-#include "driver\eeprom.h"
+#include "driver/eeprom.h"
 #include "misc.h"
 #include "radio.h"
-#include "driver\system.h"
-#include "driver\st7565.h"
+#include "driver/system.h"
+#include "driver/st7565.h"
 #include "settings.h"
-#include "driver\systick.h"
-#include "ui\helper.h"
-#include "ui\ui.h"
-#include "ui\main.h"
+#include "driver/systick.h"
+#include "ui/helper.h"
+#include "ui/ui.h"
+#include "ui/main.h"
 #include "font.h"
 #include "functions.h"
-#include "app\app.h"
+#include "app/app.h"
 
 #define _MAX_READ_CH_ATTRIBUTES 7
 #define COMBUFF_USE_SEEK_RSSI   01
