--- ui/menu.h
+++ ui/menu.h
@@ -119,6 +119,7 @@
 	MENU_F_CALI,  // reference xtal calibration
 #endif
 	MENU_BATCAL,  // battery voltage calibration
+	MENU_DBMCAL,  // Sarsat_UV-K1-5_RP2040: correction dBm par bande (dBmCorrTable)
 	MENU_F1SHRT,
 	MENU_F1LONG,
 	MENU_F2SHRT,
@@ -150,7 +151,6 @@
 extern const char        gSubMenu_D_RSP[4][11];
 #endif
 
-extern const char gSubMenu_LIVESEEK[3][6];
 
 extern const char* const gSubMenu_PTT_ID[5];
 extern const char        gSubMenu_PONMSG[4][8];
