--- ui/menu.c
+++ ui/menu.c
@@ -50,7 +50,6 @@
 	{"TxOffs", VOICE_ID_TX_OFFSET_FREQUENCY,           MENU_OFFSET        }, // was "OFFSET"
 	{"W/N",    VOICE_ID_CHANNEL_BANDWIDTH,             MENU_W_N           },
 
-	{"Live.S", VOICE_ID_CHANNEL_BANDWIDTH,             MENU_LIVESEEK      }, //by KD8CEC
 
 	{"Scramb", VOICE_ID_SCRAMBLER_ON,                  MENU_SCR           }, // was "SCR"
 	{"BusyCL", VOICE_ID_BUSY_LOCKOUT,                  MENU_BCL           }, // was "BCL"
@@ -147,6 +146,7 @@
 	{"FrCali", VOICE_ID_INVALID,                       MENU_F_CALI        }, // reference xtal calibration
 #endif
 	{"BatCal", VOICE_ID_INVALID,                       MENU_BATCAL        }, // battery voltage calibration
+	{"DbmCal", VOICE_ID_INVALID,                       MENU_DBMCAL        }, // Sarsat_UV-K1-5_RP2040: correction dBm par bande
 	{"BatTyp", VOICE_ID_INVALID,                       MENU_BATTYP        }, // battery type 1600/2200mAh
 	{"Reset",  VOICE_ID_INITIALISATION,                MENU_RESET         }, // might be better to move this to the hidden menu items ?
 
@@ -237,12 +237,6 @@
 	"NAME\n+\nFREQ"
 };
 
-const char gSubMenu_LIVESEEK[3][6] =
-{
-	"NONE",
-	"LIVE",
-	"LIVE+"
-};
 
 
 #ifdef ENABLE_ALARM
@@ -381,6 +375,15 @@
 #ifdef ENABLE_BLMIN_TMP_OFF
 	{"BLMIN\nTMP OFF",  ACTION_OPT_BLMIN_TMP_OFF}, 		//BackLight Minimum Temporay OFF
 #endif
+#ifdef ENABLE_SARSAT
+	{"SARSAT",			ACTION_OPT_SARSAT},
+#endif
+#ifdef ENABLE_APRS
+	{"APRS",			ACTION_OPT_APRS},
+#endif
+#ifdef ENABLE_SONDE
+	{"SONDE",			ACTION_OPT_SONDE},
+#endif
 };
 
 const uint8_t gSubMenu_SIDEFUNCTIONS_size = ARRAY_SIZE(gSubMenu_SIDEFUNCTIONS);
@@ -735,10 +738,6 @@
 			strcpy(String, gSubMenu_MDF[gSubMenuSelection]);
 			break;
 
-		case MENU_LIVESEEK:
-			strcpy(String, gSubMenu_LIVESEEK[gSubMenuSelection]);
-			break;
-
 		case MENU_RP_STE:
 			if (gSubMenuSelection == 0)
 				strcpy(String, "OFF");
@@ -852,6 +851,10 @@
 			break;
 		}
 
+		case MENU_DBMCAL:  // Sarsat_UV-K1-5_RP2040
+			sprintf(String, "%d", gSubMenuSelection);
+			break;
+
 		case MENU_BATTYP:
 			strcpy(String, gSubMenu_BATTYP[gSubMenuSelection]);
 			break;
